features/tweet.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Tweet
+
+  Scenario: Tweet Creation
```

Comment:
Just looked at the style guide, and it seems like we're going the other way on this with steps more like the ones you wrote. Go with that!